frontend/src/metabase/selectors/app.ts

```diff
@@ -49,6 +49,76 @@ export const getIsAdminApp = createSelector([getRouterPath], path => {
   return path.startsWith("/admin/");
 });
 
+export const getIsCollectionPathVisible = createSelector(
+  [getQuestion, getDashboard, getRouterPath, getIsEmbedded, getEmbedOptions],
+  (question, dashboard, path, isEmbedded, embedOptions) => {
+    if (isEmbedded && !embedOptions.breadcrumbs) {
+      return false;
+    }
+
+    return (
+      ((question != null && question.isSaved()) || dashboard != null) &&
+      PATHS_WITH_COLLECTION_BREADCRUMBS.some(pattern => pattern.test(path))
+    );
+  },
+);
+
+export const getIsQuestionLineageVisible = createSelector(
+  [getQuestion, getOriginalQuestion, getRouterPath],
+  (question, originalQuestion, path) =>
+    question != null &&
+    !question.isSaved() &&
+    originalQuestion != null &&
+    !originalQuestion.isDataset() &&
+    PATHS_WITH_QUESTION_LINEAGE.some(pattern => pattern.test(path)),
+);
```

Comment:
These 2 selectors are moved up since I modify other selectors to rely on these 2.